src/main/java/com/synopsys/integration/alert/channel/jira/common/model/AlertJiraIssueProperties.java

```diff
@@ -26,6 +26,7 @@
 
 public class AlertJiraIssueProperties extends AlertSerializableModel {
     private String provider;
+    private String providerUrl;
```

Comment:
When merging this you will need to update the isuetracker-jira library to include the provider URL as well since this is now an issue property.  The class will be the JiraIssueSearchProperties class. Also will this need to be added to the Jira Plugin that includes the feature to migrate issues.  Add the Black Duck Server URL as the provider URL in the Jira Plugin migration issue.